deployments/add-ons/storage/efs/dynamic-provisioning/sc.yaml

```diff
@@ -7,6 +7,8 @@ mountOptions:
   - tls
 parameters:
   directoryPerms: '700'
+  gid: <YOUR_GID>
```

Comment:
Could you set these to default values, else our tests will fail.

Reply:
Yes no problem 👍

Reply:
Hi @AlexandreBrown Any updates on this change, we can merge this PR after that.

Reply:
Hello @mbaijal , sorry for the delay, I will try to get it done on Friday.

Reply:
@mbaijal Sorry for the delay, it should be good now